client/src/main/java/org/asynchttpclient/netty/NettyResponse.java

```diff
@@ -179,6 +179,7 @@ public ByteBuffer getResponseBodyAsByteBuffer() {
         for (HttpResponseBodyPart part : bodyParts)
             target.put(part.getBodyPartBytes());
 
+        target.rewind();
```

Comment:
does it matter? I don't mind to change to flip, but it's doesn't do anything special in this context - it's updates the state of limit to be the same as already defined (which is the buffer length), while rewind just sets back to the index to the beginning.